<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsWidget2FA</class>
 <widget class="QWidget" name="ReportsWidget2FA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReportsWidget2FA</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="mfaTableView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="downloadDirectoryError">
       <property name="text">
        <string>Downloading the 2FA Directory Failed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="downloadingDirectory">
       <property name="text">
        <string>Downloading 2FA Directory (provided by 2fa.directory)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="noNetwork">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>This build of KeepassXC doesn't have networking functions. Networking is required to provide 2FA suggestions</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hintLabel">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Double-click entries to edit them</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Filter</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="supportedFilter">
          <property name="text">
           <string>Supported</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="notSupportedFilter">
          <property name="text">
           <string>Not Supported</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="enabledFilter">
          <property name="text">
           <string>Enabled</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="disabledFilter">
          <property name="text">
           <string>Disabled</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
